docs/platforms/android/index.mdx

```diff
@@ -12,36 +12,106 @@ keywords:
   - ndk
 ---
 
-<PlatformContent includePath="getting-started-primer" />
+On this page, we get you up and running with Sentry's Android SDK, automatically reporting errors and exceptions in your application.
 
-On this page, we get you up and running with Sentry's SDK.
+<Note>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
 
-Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+</Note>
 
 ## Install
 
-Sentry captures data by using an SDK within your application's runtime.
+<OnboardingOptionButtons
+  options={[
+    'error-monitoring',
+    'performance',
+    'profiling',
+  ]}
+/>
 
-<PlatformContent includePath="getting-started-install" />
+Sentry captures data by using an SDK within your application's runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.
 
-## Configure
+In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](https://docs.sentry.io/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](https://docs.sentry.io/product/profiling/). To enable tracing and/or profiling, click the corresponding checkmarks to get the code snippets.
```

Comment:
```suggestion
In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](https://docs.sentry.io/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](https://docs.sentry.io/product/profiling/). To enable tracing and/or profiling, click the corresponding checkmarks to customize the code snippets.
```

Reply:
IMO both makes sense - let's keep `get` for now as I copied that verbatim from Liza's suggestion on the Apple docs PR.